api/swagger-spec/v1.json

```diff
@@ -11540,6 +11540,14 @@
      "default": {
       "type": "any",
       "description": "default values on this kind by resource name if omitted"
+     },
+     "defaultRequest": {
+      "type": "any",
+      "description": "default resource requests values on this kind by resource name if omitted"
```

Comment:
I saw your discussion with @bgrant0607 later about not changing Default to DefaultLimit, but can you at least update the description of "default" above to "default resource limit values on this kind..." ? Or is that considered an API change?